src/main/scala/chisel3/tester/experimental/backends/verilator/CommandEditor.scala

```diff
@@ -0,0 +1,100 @@
+// See LICENSE for license details.
```

Comment:
This was code specifically written for verilator based on user feedback from Steve Burns. It might belong somewhere more generally in the long run, but I don't think it is correct to call it *legacy*